webhooks-extension/pkg/endpoints/webhook_test.go

```diff
@@ -104,7 +107,7 @@ func TestGetOpenshiftServiceDashboardURL(t *testing.T) {
 	dashboard := r.getDashboardURL(installNs)
 
 	if dashboard != "http://fake-openshift-dashboard:1234/v1/namespaces/default/endpoints" {
-		t.Errorf("Dashboard URL not http://fake-dashboard:1234/v1/namespaces/default/endpoints.  URL was %s", dashboard)
+		t.Errorf("Dashboard URL not http://ake-openshift-dashboard:1234/v1/namespaces/default/endpoints.  URL was %s", dashboard)
```

Comment:
Biggest nit ever but don't like mistakes as if it fails users may be mislead 🏳️ can you please add what it should be (missing f?) and then I'll approve?

Reply:
Also please fix the typo in the commit message, let's go with

`Always use the dashboard service for webhooks extension`

Reply:
fixed both

Reply:
Did you fix up the commit message? I still see

```
[whebhooks-extension] Always use the dashboard service for webhooks e…

…xtension
```

I think you need a git commit --amend and force push

Reply:
you want that i remove the [whebhooks-extension] prefix ?

Reply:
better now ?